docs/api/navigation-history.md

```diff
@@ -0,0 +1,26 @@
+## Class: NavigationHistory
+
+> Handles session history, maintaining a list of navigation entries for backward and forward navigation.
+Process: [Main](../glossary.md#main-process)<br />
+_This class is not exported from the `'electron'` module. It is only available as a return value of other methods in the Electron API._
+
```

Comment:
* Can we add a preamble to this section to explain a bit more in detail how the indexing system works? For example, it makes sense for me intuitively that the first available navigation history element to be at index 0 and for the latest navigation history element to be at index N. I think we should make that back/forth order more explicit in these docs.

* If people want to gather an array of all navigation history items, is the solution to loop through `getEntryAtIndex()` N times?

Reply:
Yes, that is the idea to loop N times. That is why I want to expose activeIndex and history length to know where they are and how many entries they can loop through. Edited for clarity.